packages/core/admin/admin/src/content-manager/components/BlocksEditor/utils/types.ts

```diff
@@ -0,0 +1,24 @@
+import { type Text, type Element, type Node, Editor } from 'slate';
+
+import { type SelectorBlockKey, selectorBlockKeys } from '../hooks/useBlocksStore';
```

Comment:
couldn't find these types defined under useBlocksStore file

Reply:
i'm moving them to the end of the file, hopefully it's easier to find